Resources/Prototypes/Traits/disabilities.yml

```diff
@@ -23,6 +23,14 @@
   name: trait-pacifist-name
   components:
     - type: Pacified
+      throwBlacklist:
```

Comment:
this is just the components blacklist so you dont need to duplicate it

Reply:
AJCM told me:
> Datafields have an init value set in the c#-side that's always used by default by the prototype unless overriden, this is good and all but it's good practice to set the value both in the c# and yaml sides for complex fields like entity whitelists for clarity. So, what i wanted you to do was to put the whitelist datafield values in the Pacified component of the Pacifist trait as well when you make it

But I see your point, maybe it shouldn’t be duplicated.

Reply:
if ajcm wants it in yaml simply make it a required field (`[DataField(required: true)` and use ` = new();` as the initializer so theres no need to maintain 2 of the same list

Reply:
Looks like it just calls `EnsureComponent`. I guess I’d lean towards removing the YAML from the trait definition since things like the Pax chemical also exist.